examples/python/mixed/reader.py

```diff
@@ -1,38 +1,44 @@
+"""Reader without using the wait call."""
 ##############################################################################
 # Copyright (c) 2005-2015 Real-Time Innovations, Inc. All rights reserved.
 # Permission to modify and use for internal purposes granted.
 # This software is provided "as is", without warranty, express or implied.
 ##############################################################################
 
-import sys
-import os
-filepath = os.path.dirname(os.path.realpath(__file__))
-sys.path.append(filepath + "/../../../");
-import time
+from sys import path as sysPath
+from os import path as osPath
+from time import sleep
 import rticonnextdds_connector as rti
 
+filepath = osPath.dirname(osPath.realpath(__file__))
+sysPath.append(filepath + "/../../../")
 
-connector = rti.Connector("MyParticipantLibrary::Zero",filepath + "/../Mixed.xml");
-input = connector.getInput("MySubscriber::MySquareReader");
+connector = rti.Connector("MyParticipantLibrary::Zero",
+                          filepath + "/../Mixed.xml")
+inputDDS = connector.getInput("MySubscriber::MySquareReader")
```

Comment:
we want to keep input (or output) as names for the variables. This are the names we use in the docuement and they represent a concept. So i wouldn't rename them. The example is here to teach how to use connector.. so naming is important